Gemfile

```diff
@@ -47,8 +48,8 @@ group :development do
   gem 'listen'
   # Spring speeds up development by keeping your application running in the
   # background. Read more: https://github.com/rails/spring
-  gem 'spring'
-  gem 'spring-watcher-listen'
+  gem 'spring', '~> 4.1'
+ # gem 'spring-watcher-listen'
```

Comment:
if this is no longer required or wanted, then remove rather than comment out.

Reply:
will do